### PR TITLE
add descriptions for unsupervised fine-tuning

### PR DESCRIPTION
**Pull request by guenthermi**
This PR contains a description on how to use finetuner and tutorials to fine-tune SBERT and CLIP models in an unsupervised way: 
The overall description is placed in the 13th Kata and the code is placed in a new `finetuner/` directory together with a README which describes how to execute it.

### CODE REVIEW

**Comment by guenthermi** [comment]
I did not include evaluation results. I also have some scripts on my machine to evaluate the models, but especially for CLIP they are a bit slow without a GPU machine

File: finetuning/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+## Setup the Environment
+
+Finetuner runs with Python.
+Accordingly, you need a Python environment to run it.
+We recommend you to create also a virtual environment by running:
+``python3 -m venv venv``
+Afterward, you can activate the environment by running:
+``source venv/bin/activate``
+To install the dependencies, to run the fine-tuning exaples, run
```
Comment (atarora) [inline finetuning/README.md:16]: typo 'examples'

File: finetuning/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Since this dataset does not come with a set of queries (which can usually be extracted from a query log), we generate some queries.
+To download them one can execute the following command:
+```
+wget https://finetuner-ecommerce-experiment.s3.eu-central-1.amazonaws.com/generated-queries.jsonl -P sbert_unsupervised/
+```
+To extract queries and documents and transform them into Finetuner's preferred format for finetuner execute the following command*:
+```
+python3 sbert_unsupervised/data_preparation.py
+```
+*While running the script, a browser window will open up and ask you to log into your Jina account. If you don't have one, you need to sign up to run the script.
```
Comment (atarora) [inline finetuning/README.md:32]: How is running the script connected to jina account, could you elaborate?

File: katas/013_finetuning_embedding_models.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+4. Integrate the Model: Once the fine-tuning is complete, you can download the model, which can be integrated as a replacement into Chorus.
+
+### Unsupervised Training of Text-to-Text Embedding Models
+
+We begin by fine-tuning a pure text embedding model using an unsupervised approach.
+In this approach, data annotation is not required.
+Instead, the finetuner framework generates a dataset using a given corpus, such as our product database, along with a set of queries that can be extracted from a query log.
+Since there is no query log for the demo database, we provide generated [queries](https://finetuner-ecommerce-experiment.s3.eu-central-1.amazonaws.com/generated-queries.jsonl) for download to run the demo code.
+
+To establish relationships between queries and documents, we employ a pseudo labeling approach, similar to the one described in this paper.
```
Comment (atarora) [inline katas/013_finetuning_embedding_models.md:41]: is the paper link missing here?

**Review by atarora — APPROVED** [review]
Looks good !! Thanks for your contribution @guenthermi !!

**Pull request merged**